tox.ini

```diff
@@ -25,3 +25,16 @@ deps =
 
 commands =
     flake8 lona tests
+
+[testenv:mypy]
+deps =
+    mypy
+
+commands =
+    mypy --config-file tox.ini lona
+
+
+[mypy]
+
+[mypy-rlpython.*]
```

Comment:
What does this do? Why is this necessary for rlpython and not the other dependencies?

Reply:
Added comment to `mypy.ini`
```
Avoid error: Skipping analyzing "rlpython": found module but no type hints or library stubs
```

Reply:
Here is a run without this config: https://github.com/lona-web-org/lona/runs/3514862356?check_suite_focus=true

Reply:
Ah ok. Do we have to black-list all third party-libraries that produce mypy warnings or can we white-list the lona module instead?

Reply:
I'd say that black list is better. It is more strict. But I can change if you want.

Reply:
A black list has to be maintained and the listed modules can change with every test-run. If aiohttp merges a module that throws mypy warnings, our linting stage fails on the next run.

Reply:
> If aiohttp merges a module that throws mypy warnings, our linting stage fails on the next run.

As far as I understand it works differently. And that may not happen. However I'm not 100% sure.

Official doc [recommends](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-type-hints-for-third-party-library) to avoid this.
> To suppress all missing import errors for all libraries in your codebase, [...] set the `ignore_missing_imports` config file option to `True` in the global section of your mypy config file.
> [...]
> We recommend using this approach only as a **last resort**: it’s equivalent to adding a # type: ignore to all unresolved imports in your codebase.

Reply:
@maratori That convinces me. Thanks for researching this!